docs: State that authenticationKeyManager is removed in 4.0 (#748)
serverpod/serverpod_docs: docs/11-upgrading/01-upgrade-to-four.md

  serverpod_auth_shared_flutter: 4.0.0-beta.1  # in the Flutter package
```

> The `authenticationKeyManager` parameter on the generated `Client` is gone in 4.0. Assign the key manager to the `authKeyProvider` field instead:

Reviewer: Suggestion: "gone" reads casual for a breaking change in an upgrade guide, and the PR title already uses the plainer word.

```suggestion
The `authenticationKeyManager` parameter on the generated `Client` was removed in 4.0. Assign the key manager to the `authKeyProvider` field instead:
```
